src/dotnet-new3/Program.cs

```diff
@@ -23,6 +23,11 @@ public static class Program
 
         public static int Main(string[] args)
         {
+            if (Console.IsOutputRedirected)
+            {
+                Console.OutputEncoding = System.Text.Encoding.UTF8;
```

Comment:
Maybe add some comment why we are doing this for future archaeologist(me in 6 months).